### DISCUSSION TITLE
Bromite release tag / version choice

### DISCUSSION THREAD

**Discussion by chirayudesai** [bromite/bromite#1998] [Builds]
Apologies if this has been documented elsewhere already, but I couldn't find it.

Is there a particular reason for the choice of the upstream Chromium tags Bromite uses for releases?

Current Bromite stable version: 100.0.4896.135
Google's stable: 100.0.4896.127 (per https://chromiumdash.appspot.com/releases?platform=Android and https://chromereleases.googleblog.com/2022/04/chrome-for-android-update_01436352408.html)

Previous Bromite version was 100.0.4896.92, whereas for Google it was 100.0.4896.88

Just curious.

**Comment by csagan5** [comment]
> Is there a particular reason for the choice of the upstream Chromium tags Bromite uses for releases?

Not really; only criteria is that the extra commits are not introducing any big change for the Android platform (and they usually never are). Bromite version will usually be a bit higher than Chromium but most of the extra commits are bogus (bot commits) or apply to other platforms (desktop) or making small changes.
